public/data/Aditya-150.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Aditya Prasad Mohanty",
+    "type": "personal",
+    "bio": "Open Source Enthusiast | 🕷🕸🕷",
+    "avatar": "https://github.com/Aditya-150.png",
+    "links": [
+        {
+            "name": "Follow me on Github",
```

Comment:
I recommend capitalizing the `h` here. What do you think?
```suggestion
            "name": "Follow me on GitHub",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
> I recommend capitalizing the `h` here. What do you think?
> 
> __ [![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966)](https://semasoftware.com/gh) [ ![sema-logo](https://camo.githubusercontent.com/843687ee2a84b8cc104a1e4297c26c9db3d3f710080228887db6338b4550a240/68747470733a2f2f6170702e73656d61736f6674776172652e636f6d2f696d672f73656d612d747261792d6c6f676f2e676966) ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** ❓ I have a question

Ya, sure . I should have done that. Thanks for your recommendation.